subprojects/ear/src/main/java/org/gradle/plugins/ear/internal/DefaultEarPluginConvention.java

```diff
@@ -45,6 +47,11 @@ public DefaultEarPluginConvention(FileResolver fileResolver, ObjectFactory objec
         deploymentDescriptor.readFrom(appDirName + "/META-INF/" + deploymentDescriptor.getFileName());
     }
 
+    @Override
+    public TypeOf<?> getPublicType() {
+        return TypeOf.typeOf(EarPluginConvention.class);
```

Comment:
⭕️ static import `typeOf` everywhere?

Reply:
`TypeOf` is already imported so I thought it would add noise for no real benefit